src/libsync/theme.cpp

```diff
@@ -341,10 +341,9 @@ QString Theme::gitSHA1() const
 QString Theme::about() const
 {
     QString devString;
-    devString = tr("<p>Version %1. For more information please visit <a href='%2'>%3</a>.</p>")
+    devString = tr("<p>Version %1. For more information please click <a href='%2'>here</a>.</p>")
               .arg(MIRALL_VERSION_STRING)
-              .arg("http://" MIRALL_STRINGIFY(APPLICATION_DOMAIN))
-              .arg(MIRALL_STRINGIFY(APPLICATION_DOMAIN));
+              .arg("http://" MIRALL_STRINGIFY(APPLICATION_DOMAIN));
```

Comment:
I would simply hardcode "https://www.nextcloud.com" here.
And on branding system we can replace this with help-url.

Reply:
But isn't APPLICATION_DOMAIN that will be replaced?